.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: vim
+rvm:
+  - 1.8.7 # Use stock
+  - 1.9.2 # Build latest
```

Comment:
Ahh, this is useful then. How about using multiple `env` instead? http://docs.travis-ci.com/user/build-configuration/#The-Build-Matrix A bit more elegant since we don't care about Ruby, and easier to retrieve in the if checks.